app/docs/controls.docs.tsx

```diff
@@ -57,6 +58,22 @@ const ControlsDoc = () => {
     </ReactSpecimen>
   )}
 
+## OnOffControlTab
+OnOffControlTab (and OnOffControlTabField) are elements which are supposed to be used on the left panel in the app as category "switches"
+(like for BaseLayer in case of maps or InteractiveFilters for... interactive filters). They display either "on" or "off" to indicate component state.
+
+
+  ${(
+    <ReactSpecimen span={2}>
+      <OnOffControlTab
+        value="Test"
+        label={<span>Test</span>}
+        icon="settings"
+        onClick={() => {}}
+      />
+    </ReactSpecimen>
+  )}
```

Comment:
Thanks !

Reply:
I think that we should refactor the docs to make them more consistent... Also `Dataset Browsing` docs do not work, but I think it would make sense to tackle this + adding more docs & improving current ones as a future ~ 1-day task? :)

Reply:
Good point: https://github.com/visualize-admin/visualization-tool/issues/306